test-suite/src/function/length.rs

```diff
@@ -0,0 +1,52 @@
+use gluesql_core::prelude::Value;
+
+use {crate::*, gluesql_core::prelude::Payload};
+
+test_case!(length, async move {
+    test! {
+        name: "test length with string",
+        sql: "SELECT LENGTH('Hello.');",
+        expected: Ok(Payload::Select { labels: vec!["LENGTH('Hello.')".to_owned()], rows: vec![vec![Value::U64(6)]] })
```

Comment:
Let's use the `select!` macro to write the expected part like any other test case.

Reply:
https://github.com/gluesql/gluesql/blob/29f4128d38a11848cf123c60dd6b9b2f4076cbd7/test-suite/src/basic.rs#L44-L51